modular_zubbers/code/modules/hydroponics/gene_modder.dm

```diff
@@ -255,31 +256,34 @@
 	popup.open()
 
 
-/obj/machinery/plantgenes/Topic(href, list/href_list)
+/obj/machinery/plantgenes/Topic(var/href, var/list/href_list)
```

Comment:
will fail linters, no need to declare var in proc args

Reply:
Oh, that's why linters is failing? Weird, I'll change it right away!